src/python/bcc/__init__.py

```diff
@@ -1290,7 +1293,10 @@ def trace_fields(self, nonblocking=False):
             line = line[ts_end + 1:]
             sym_end = line.find(b":")
             msg = line[sym_end + 2:]
-            return (task, int(pid), int(cpu), flags, float(ts), msg)
+            try:
+                return (task, int(pid), int(cpu), flags, float(ts), msg)
+            except Exception as e:
+                return (task, pid, cpu, flags, ts, msg)
```

Comment:
Is the change here necessary? Even if we return in Exception to caller, it may still have exceptions due to ill-formed fields, right?

Reply:
You're right，I need to fill the return:
return ("Unknown", 0, 0, "Unknown", 0.0, "Unknown")